src/Network/PacketHandlers.cs

```diff
@@ -4849,6 +4858,7 @@ private static void MegaCliloc(ref StackDataReader p)
                         if (entity != null && !SerialHelper.IsMobile(serial))
                         {
                             entity.Name = str;
+                            entity.NameCliloc = s.Item3 > 0 ? s.Item3 : s.Item1;
```

Comment:
Just back in mind about the OPL object.
It could contain the `NameCliloc` property instead of add it to each item.

Reply:
yeah!